src/renderers/noop/ReactNoop.js

```diff
@@ -146,6 +149,10 @@ var ReactNoop = {
     ReactNoop.flushLowPri();
   },
 
+  performHighPriWork(fn: Function) {
+    NoopRenderer.performWithPriority(HighPriority, fn);
```

Comment:
> If we don't expose it to users, maybe we also shouldn't expose it to renderers?

I can imagine renderers might need at least some more granular control than a user, because there are certain scheduling tricks that can only be controlled at the renderer level. E.g. I had assumed that the deprioritization that happens when `hidden={true}` (or `style={{ display: hidden }}`) would eventually be handled by the renderer via some API, rather than being embedded in Fiber itself. Or a renderer might want to use a different priority for specific types of events? Curious what your plans are for that.